application/uitests/org.openjdk.jmc.flightrecorder.uitest/src/test/java/org/openjdk/jmc/flightrecorder/uitest/JfrThreadsPageTest.java

```diff
@@ -84,6 +93,32 @@ public void after() {
 		}
 	};
 
+	boolean selected = false;
```

Comment:
Perhaps this could be moved towards the top alongside the other variables, and the value could be returned to `false` in the `after()`

Reply:
Addressed in https://github.com/aptmac/jmc/pull/49/commits/ae6f31c61c1e5c64445508e0770d105703d2d1d5